sc-implementation/src/test/java/com/powsybl/sc/implementation/ShortCircuitBalancedTest.java

```diff
@@ -0,0 +1,740 @@
+package com.powsybl.sc.implementation;
```

Comment:
/**
 * Copyright (c) 2022, Jean-Baptiste Heyberger & Geoffroy Jamgotchian
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

Reply:
All new files.